src/test/java/com/microsoft/aad/msal4j/HttpHeaderTest.java

```diff
@@ -38,14 +44,18 @@ public void testHttpHeaderConstructor(){
     }
 
     @Test
-    public void testHttpHeaderConstructor_valuesNotSet(){
+    public void testHttpHeaderConstructor_valuesNotSet() {
 
         PublicClientApplication app = PublicClientApplication
                 .builder("client-id")
                 .build();
 
+        IAcquireTokenParameters parameters = UserNamePasswordParameters
+                .builder(Collections.singleton("scopes"), "username", "password".toCharArray())
+                .build();
+
         RequestContext requestContext = new RequestContext(
-                app, PublicApi.ACQUIRE_TOKEN_BY_AUTHORIZATION_CODE, null);
+                app, PublicApi.ACQUIRE_TOKEN_BY_AUTHORIZATION_CODE, parameters);
```

Comment:
nothing is tested here, except that this call will not throw exception

Reply:
Due to the changes in HttpHeaders, unless parameters were passed in this would lead to null ref.